gradle.properties

```diff
@@ -5,19 +5,22 @@ pluginGroup = com.github.madwareru.intellijronremix
 pluginName = RON Extended Support
 pluginVersion = 0.2.11
 
-pluginSinceBuild = 231.0
-pluginUntilBuild = 231.*
+pluginSinceBuild = 232.0
```

Comment:
Please add an entry in CHANGELOG.md about the update of the version of IntelliJ Platform supported

Reply:
Added